### PR TITLE
Update links to point to dev docs

### PR DESCRIPTION
**Pull request by joethei**
- replace submission instruction links with links to dev docs.
- removes the `plugin-review.md` file
- update the link in the plugin template to point to plugin guidelines in the dev docs
- add link to theme guidelines in theme template
- replace open source & advertisement policy with link to dev policies.

### CODE REVIEW

**Comment by ericaxu** [comment]
I feel this is pretty safe. Do you need approval from both of them?

**Comment by joethei** [comment]
No, it would be nice if Licat could take a quick look at the last one though.

**Comment by ericaxu** [comment]
> * replace open source & advertisement policy with link to dev policies.

Last one being "replace open source & advertisement policy with link to dev policies", right? If so I'll let Licat know this is blocked on him.

**Comment by joethei** [comment]
corect

File: README.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ When opening a pull request, please switch to preview mode and select the option
 
 Thanks for submitting your creations!
 
-You can find a detailed explanation for submitting your [plugin here](https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/Guides/How+to+add+your+plugin+to+the+community+plugin+list) and your [theme here](https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/Guides/How+to+add+your+theme+to+the+community+theme+store).
+You can find a detailed explanation for submitting your [plugin here](https://docs.obsidian.md/Plugins/Releasing/Submit+your+plugin) and your [theme here](https://docs.obsidian.md/Themes/App+themes/Submit+your+theme).
+
+All submissions must conform with our [developer policies](https://docs.obsidian.md/Developer+policies)
```
Comment (lishid) [inline README.md:17]: Can we make this its own heading so it's emphasized?

**Review by lishid — APPROVED** [review]

**Pull request merged**